my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
+data = fits.getdata('/Users/bostroem/Desktop/images/hubble_img.fits')
```

Comment:
It's typically good to add newlines or spaces after conceptual blocks of code. For example, here after your imports you could add a newline, and then after defining your `col` variables.